Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -496,6 +496,9 @@ void GameList::CompressISO(bool decompress)
             .append(decompress ? QStringLiteral(".gcm") : QStringLiteral(".gcz")),
         decompress ? tr("Uncompressed GC/Wii images (*.iso *.gcm)") :
                      tr("Compressed GC/Wii images (*.gcz)"));
+
+    if (dst_dir.isEmpty())
+      return;
```

Comment:
Shouldn't this be `dst_path`?

Reply:
Looks like it should. That also makes my comment void, since it's not the same thing being checked.

Reply:
Whoops, sure